src/js/view/components/nextuptooltip.js

```diff
@@ -140,7 +140,7 @@ define([
                 return;
             }
 
-            this.showNextUp = true;
+            this.showNextUp = nextUp.showNextUp;
```

Comment:
`nextUp.showNextUp` is `undefined` for playlists. 

Test with: `feature=playlist/playlist_next_up_display`.